### PR TITLE
Update cloud plugin documentation to make consistent with new metrics.

### PR DESCRIPTION
**Pull request by bryanatkinson**
Checklist (if applicable):
- [x] Docs updated

### CODE REVIEW

File: docs/plugins/google-cloud.md [hunk 1]
```diff
@@ -221,32 +221,100 @@ Common dimensions include:
...
+| genkit/ai/generate/input/audio       | Counter tracking input audio files   |
+| genkit/ai/generate/output/audio      | Counter tracking output audio files  |
+
+Each generate-level metric contains the following dimensions:
+
+| Name            | Description                                              |
+| --------------- | -------------------------------------------------------- |
+| modelName       | The name of the model being called.                      |
+| featureName     | The name of the feature being executed that lead to this |
+:                 : model being called.                                      :
```
Comment (MichaelDoyle) [inline docs/plugins/google-cloud.md:304]:
```suggestion
| featureName     | The name of the parent feature being executed.           |
```

File: docs/plugins/google-cloud.md [hunk 2]
```diff
@@ -221,32 +221,100 @@ Common dimensions include:
...
+| genkit/ai/generate/input/images      | Counter tracking input images        |
+| genkit/ai/generate/output/images     | Counter tracking output images       |
+| genkit/ai/generate/input/audio       | Counter tracking input audio files   |
+| genkit/ai/generate/output/audio      | Counter tracking output audio files  |
+
+Each generate-level metric contains the following dimensions:
+
+| Name            | Description                                              |
+| --------------- | -------------------------------------------------------- |
+| modelName       | The name of the model being called.                      |
```
Comment (MichaelDoyle) [inline docs/plugins/google-cloud.md:302]:
```suggestion
| modelName       | The name of the model.                                   |
```

File: docs/plugins/google-cloud.md [hunk 3]
```diff
@@ -221,32 +221,100 @@ Common dimensions include:
...
+| maxOutputTokens | The max output tokens parameter passed to the model      |
+| topK            | The topK parameter passed to the model                   |
+| topP            | The topP parameter passed to the model                   |
+| latencyMs       | The response time taken by the model                     |
+| status          | 'success' or 'failure' depending on whether or not the   |
+:                 : feature request succeeded.                               :
+| error           | Only set when `status=failure`. Contains the error type  |
+:                 : that caused the failure.                                 :
+| source          | The genkit source language. Eg. 'ts'.                    |
+| sourceVersion   | The version of the genkit framework used                 |
```
Comment (MichaelDoyle) [inline docs/plugins/google-cloud.md:317]:
```suggestion
| source          | The Genkit source language. Eg. 'ts'.                    |
| sourceVersion   | The Genkit framework version                             |
```

File: docs/plugins/google-cloud.md [hunk 4]
```diff
@@ -221,32 +221,100 @@ Common dimensions include:
...
+
+| Name            | Description                                              |
+| --------------- | -------------------------------------------------------- |
+| modelName       | The name of the model being called.                      |
+| featureName     | The name of the feature being executed that lead to this |
+:                 : model being called.                                      :
+| path            | The path of execution from the feature root to this      |
+:                 : action. eg. '/myFeature/parentAction/thisAction'         :
+| temperature     | The temperature parameter passed to the model            |
+| maxOutputTokens | The max output tokens parameter passed to the model      |
```
Comment (MichaelDoyle) [inline docs/plugins/google-cloud.md:308]:
```suggestion
| maxOutputTokens | The maxOutputTokens parameter passed to the model        |
```

**Comment by MichaelDoyle** [comment]
I noticed sometimes we use periods (`.`) in field descriptions, and sometimes we don't. Should just pick one method and stick to it.  :)

File: docs/plugins/google-cloud.md [hunk 5]
```diff
@@ -221,32 +221,100 @@ Common dimensions include:
...
+
+Features are the top-level entry-point to your Genkit code. In most cases, this
+will be a Genkit flow, but if you do not use flows this will be the top-most
+span in a Genkit trace.
+
+Name                    | Description
+----------------------- | ---------------------------------------------------
+genkit/feature/requests | Counter for number of requests by feature.
+genkit/feature/latency  | Histogram tracking feature execution latency in ms.
+
```
Comment (MichaelDoyle) [inline docs/plugins/google-cloud.md:234]: I wonder if we should do something like this in all the sections.

```suggestion
Name                    | Type      | Description
----------------------- | --------- | ---------------------------------------------------
genkit/feature/requests | Counter   | Number of requests.
genkit/feature/latency  | Histogram | Execution latency in ms.
```

Comment (bryanatkinson) [reply docs/plugins/google-cloud.md]: Good idea, updated!

File: docs/plugins/google-cloud.md (hunk 1 above)
Comment (bryanatkinson) [reply docs/plugins/google-cloud.md]: Updated.

File: docs/plugins/google-cloud.md [hunk 6]
```diff
@@ -221,32 +221,85 @@ Common dimensions include:
...
+| genkit/ai/generate/output/images     | Counter   | Output images                              |
+| genkit/ai/generate/input/audio       | Counter   | Input audio files                          |
+| genkit/ai/generate/output/audio      | Counter   | Output audio files                         |
+
+Each generate-level metric contains the following dimensions:
+
+| Name            | Description                                                                                          |
+| --------------- | ---------------------------------------------------------------------------------------------------- |
+| modelName       | The name of the model                                                                                |
+| featureName     | The name of the feature being executed that lead to this model being called                          |
```
Comment (MichaelDoyle) [inline docs/plugins/google-cloud.md:292]:
```suggestion
| featureName     | The name of the parent feature being executed                          |
```

File: docs/plugins/google-cloud.md [hunk 7]
```diff
@@ -221,32 +221,85 @@ Common dimensions include:
...
+
+| Name          | Description                                                                                          |
+| ------------- | ---------------------------------------------------------------------------------------------------- |
+| name          | The name of the action being tracked                                                                 |
+| featureName   | The name of the parent feature being executed                                                        |
+| path          | The path of execution from the feature root to this action. eg. '/myFeature/parentAction/thisAction' |
+| status        | 'success' or 'failure' depending on whether or not the action succeeded                              |
+| error         | Only set when `status=failure`. Contains the error type that caused the failure                      |
+| source        | The genkit source language. Eg. 'ts'                                                                 |
+| sourceVersion | The version of the genkit framework used                                                             |
```
Comment (MichaelDoyle) [inline docs/plugins/google-cloud.md:266]:
```suggestion
| source        | The Genkit source language. Eg. 'ts'                                                                 |
| sourceVersion | The Genkit framework version                                                             |
```

File: docs/plugins/google-cloud.md [hunk 8]
```diff
@@ -221,32 +221,85 @@ Common dimensions include:
...
+
+Each feature-level metric contains the following dimensions:
+
+| Name          | Description                                                                      |
+| ------------- | -------------------------------------------------------------------------------- |
+| name          | The name of the feature. In most cases, this is the top-level genkit flow        |
+| status        | 'success' or 'failure' depending on whether or not the feature request succeeded |
+| error         | Only set when `status=failure`. Contains the error type that caused the failure  |
+| source        | The genkit source language. Eg. 'ts'                                             |
+| sourceVersion | The version of the genkit framework used                                         |
```
Comment (MichaelDoyle) [inline docs/plugins/google-cloud.md:243]:
```suggestion
| source        | The Genkit source language. Eg. 'ts'                                             |
| sourceVersion | The Genkit framework version                                         |
```

**Review by MichaelDoyle — APPROVED** [review]

File: docs/plugins/google-cloud.md [hunk 9]
```diff
@@ -221,32 +221,100 @@ Common dimensions include:
...
-| genkit/flow/latency  | flow_name                            |
 
 ### Action-level metrics
 
-| Name                   | Dimensions                           |
-| ---------------------- | ------------------------------------ |
-| genkit/action/requests | flow_name, error_code, error_message |
-| genkit/action/latency  | flow_name                            |
+Actions represent a step of genkit execution. Every Genkit span in a trace is
+considered an action and will have the following metrics tracked:
```
[the remark this answers is not in the document]
Comment (MichaelDoyle) [reply docs/plugins/google-cloud.md]: Ok, so is it safe to say these are "AI Monitoring actions" and not "Genkit actions"?

[the remark this answers is not in the document]
Comment (MichaelDoyle) [reply docs/plugins/google-cloud.md]: New verbiage seems reasonable.

[the remark this answers is not in the document]
Comment (bryanatkinson) [reply docs/plugins/google-cloud.md]: I wouldn't call them "AI Monitoring actions", they are just Genkit action spans, but I'm having a hard time defining exactly what that is. I think the current verbiage is accurate, but if it seems too vaque we can make this a little more defined.

**Pull request merged**